netcode.c

```diff
@@ -3762,6 +3821,7 @@ void netcode_server_disconnect_client_internal( struct netcode_server_t * server
     netcode_assert( client_index < server->max_clients );
     netcode_assert( server->client_connected[client_index] );
     netcode_assert( !server->client_loopback[client_index] );
+    //netcode_assert( server->skillz_matches );
```

Comment:
style thing - but you can remove commented out code, it'll be saved in git history anyway